Assignments/stu6/assignments.py

```diff
@@ -6,11 +6,16 @@
 
 
 def ex1():
-    print("TODO: Ex. 1...")
+    hello_world = range(3)
+    for i in hello_world:
+        print("Hello World from Python")
 
 
 def ex2():
-    print("TODO: Ex. 2...")
```

Comment:
Ok, good, this is better, I don't see any odd files.  However we need to put this in a function.
```
def ex2():
    array = [1, 2, 3]
    result = array_to_string(array)
    print(result)
```

Reply:
pushed the update